PREPARATION.md

```diff
@@ -1,101 +1,79 @@
-# Preparing to deploy your 1Password SCIM bridge
+# Prepare to deploy 1Password SCIM Bridge
 
-This guide will help you prepare to deploy your 1Password SCIM bridge.
+*Learn how to prepare your environment and 1Password account to use 1Password SCIM Bridge for automated user provisioning.*
 
-## High-Level Overview
+## Overview
 
-The SCIM bridge relies on the [SCIM protocol](http://www.simplecloud.info/), and acts as an intermediary between your Identity Provider - Azure Active Directory, Okta, and others - and your 1Password instance.
+1Password SCIM Bridge uses [System for Cross-domain Identity Management](https://en.wikipedia.org/wiki/System_for_Cross-domain_Identity_Management) (SCIM) to act as an intermediary between your identity provider, such as Okta or Azure Directory, and your 1Password instance. It allows you to centralize user management to your identity provider so you can automatically provision and manage users and groups in 1Password based on assignments in your identity provider.
 
-It allows for automatic provisioning and deprovisioning of your 1Password user accounts and groups based on what accounts and groups you have assigned in your Identity Provider, providing a way to centralize your organization's 1Password account with other services you may be using.
+> **Note**
+> 
+> In code and the command line, "1Password" is often referred to as `op`. For example, `op-scim`.
 
-### Technical Components
+### Technical components
 
-For general deployment, the SCIM bridge requires three things to function correctly:
+For general deployment, the SCIM bridge requires these components:
 
-* the `op-scim` service itself
+* the `op-scim` service
 * a [redis](https://redis.io/) cache
-* a domain name (example: `op-scim-bridge.example.com`)
+* a domain name (for example, `op-scim-bridge.example.com`)
 
 ### DNS record
 
-You will need to be able to create a DNS record with the SCIM bridge domain name decided. However, you'll need to have the IP address of the host, which necessitates deploying the SCIM bridge first, unless you have a static IP already assigned. Follow the steps in each respective deployment guide on when to finish setting up your DNS record.
+You'll need to be able to create a DNS record with the SCIM bridge domain name you want to use. However, you'll need to have the IP address of the host, so the bridge will need to be deployed first, unless you have a static IP already assigned. Follow the steps in the deployment example for guidelines on when to finish setting up your DNS record.
```

Comment:
The order of operations felt a little confusing to me here. If most people are going to need to deploy first, and the main takeaway is that you'll need to create the record and most people will need to do it at a specific time, would it be too drastic to simplify it to something like this? I assume this wouldn't negatively impact people who already have their IP address assigned, though if you decide to go this route it'd be good to double check with a technical reviewer.

"You'll need to create a DNS record with your SCIM bridge domain name after the bridge is deployed. Follow the steps in each respective deployment guide to finish setting up your DNS record at the appropriate time."

Reply:
I think that makes sense!

```suggestion
You'll need to create a DNS record with your SCIM bridge domain name after the bridge is deployed. Follow the steps in each respective deployment guide to finish setting up your DNS record at the appropriate time.
```